### ISSUE TITLE
homebridge-printer broken in new versions (last working is 1.0.3)

### ISSUE THREAD

**Issue by abry-cz** [SeydX/homebridge-printer#21]
Last version that is still working for me is 1.0.3

Printer switch never turns on with newer versions (1.1.x).

Log:
[7. 4. 2022 23:34:12] [PrinterPlatform] Printer: {"version":"2.0","statusCode":"successful-ok","id":55443581,"operation-attributes-tag":{"attributes-charset":"utf-8","attributes-natural-language":"en-us","printer-uri":"http://10.0.1.174:631/ipp/print"},"printer-attributes-tag":{"queued-job-count":0,"marker-names":"Black Toner_S/N_:CRUM-210617A7433","marker-levels":30,"printer-make-and-model":"HP Laser MFP 135w","printer-is-accepting-jobs":true,"printer-state":"idle","printer-up-time":31976}}

My config:
{
            "platform": "PrinterPlatform",
            "debug": true,
            "printer": [
                {
                    "name": "Printer",
                    "address": "http://10.0.1.174:631/ipp/print",
                    "polling": 10,
                    "marker": true,
                    "switchType": "SWITCH",
                    "manufacturer": "HP",
                    "model": "HP Laser MFP 135w",
                    "serialNumber": xxxxxxxxxx"
                }
            ]
        },

**Comment by r2shyyou** [comment]
indeed, reverting to 1.0.3 enables the printer switch to function correctly

fyi @seydx

**Comment by CyberGiant443** [comment]
Are any updates coming to address iOS 16.x? I can only use 1.0.3 and that version doesn’t seem to give me marker levels. Also is there anyway to disable the contact sensor for printing from showing up as a characteristic? The room I have this in has a lot of other characteristics that make it crowded at the top. see attacked pic. Thank you very much. Appreciate your work on this. 
![E13BF963-C294-42D9-882F-D81C9FBCFA3E](https://user-images.githubusercontent.com/121060157/222970605-a7e30aa4-c1bb-4f3b-8e15-38e37418371e.jpeg)

**Comment by jbbmb** [comment]
Commenting this to ping @seydx since I suffer from the same problems on this thread 👀
